src/main/java/com/contrastsecurity/http/ServerEnvironment.java

```diff
@@ -20,21 +20,27 @@
  * #L%
  */
 
-import lombok.Getter;
-
 public enum ServerEnvironment {
   DEVELOPMENT("Development"),
   QA("QA"),
   PRODUCTION("Production");
 
-  @Getter private String label;
+  private final String label;
+
+  public String getLabel() {
+    return label;
+  }
 
   ServerEnvironment(String label) {
     this.label = label;
   }
 
+  public String toURIString() {
+    return this.label.toUpperCase();
+  }
+
   @Override
   public String toString() {
-    return this.label.toLowerCase();
+    return toURIString();
```

Comment:
I assume that GSON calls `toString()` when serializing enums, and that's why this fixes the issue. Is that right?

Reply:
The TraceFilterForm and ApplicationFilterForm calls toString() when creating the uri parameters.

```Java
if (environments != null && !environments.isEmpty()) {
      filters.add(
          "environments="
              + environments.stream().map(Object::toString).collect(Collectors.joining(",")));
    }
```